pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,12 +54,13 @@ type Response struct {
 
 type Client struct {
 	*client.Client
-	Request  Request
-	Response Response
-	Error    bool
-	Created  bool
-	server   *httptest.Server
-	handler  *utiltesting.FakeHandler
+	Clientset *release_1_1.Clientset
```

Comment:
In practice, since this is new, I think we can just make an announcement when we think it's ready, and we'll tell people which ones are locked and which ones are still in flux.